packages/file/package.json

```diff
@@ -20,14 +20,15 @@
   "browser": {
     "./src/lib.node.js": "./src/lib.js"
   },
+  "types": "./dist/src/lib.d.ts",
   "exports": {
     ".": {
+      "types": "./dist/src/lib.d.ts",
```

Comment:
Unsure why this and `form-data` didn't have `types` as a conditional export but added it to align with the other packages.